gg/src/gg_conf.erl

```diff
@@ -245,7 +245,7 @@ get_override_conf(Conf) ->
 
 %% We don't set cacertfile in ssl options. In order for this to take
 %% in emqx_conf_cli:load_config, we must remove it from the configuration map.
-%% Setting cacertfile to null, undefined, or empty string does not work (as of EMQX 5.1.2)
+%% Setting cacertfile to null, undefined, or empty string does not work (as of EMQX 5.1.3)
```

Comment:
Is this supposed to be changed?

Reply:
Looking back, it seems that the comment should have stayed as 5.1.1. Will update